src/components/Common/TaxInputs/TaxInputs.tsx

```diff
@@ -85,7 +87,13 @@ export function SelectInput({ question, requirement, isDisabled = false }: EmpQ
   )
 }
 
-export function TextInput({ question, requirement, isDisabled = false }: EmpQ | CompR) {
+export function TextInput({
```

Comment:
Doesen't matter, but if you have a text input, and it allows you to set it types to text.... is it then just an input? 🤔

Reply:
Good question! I would've said no before this PR but now i need that functionality 😂 

we may have to rethink it if it throws a curveball for adapter folks. but for our purposes here it works well to restrict the input to numeric characters only while providing string output

if we do have to get away from this we'll need to find a way to use the number input and then convert the value to a string for hook form

Reply:
Yeah its fine for now especially since these are Tax Inputs. Just had the thought that we were pulling an `ATM Machine` faux pas